### DISCUSSION TITLE
License regarding the images in samples gallery

### DISCUSSION THREAD

**Discussion by ButzYung** [vladmandic/human#242] [Q&A] [answered]
I would like to use some of the images (mainly those 3D renders) under the `human` samples folder as a fallback input when there is no camera available, but I am cocerned about the license of those images, since some of them have the Daz Studio label. I wonder if they are royalty free / under MIT license like `human`?

**Comment by vladmandic** [comment, marked as answer]
good question - in general, daz3d license is quite permissive for 2d renders but it's still a proprietary license which is not ideal  

there are plenty of good cgi renders under cc (creative-commons) license which would be more of an mit license equivalent for digital art - i'll swtich sample images (good source is deviantart.com)

btw, all non-cgi images are actually mine, so no licensing issues at all :)

**Comment by vladmandic** [comment]
i've just updated samples and now all images are either mine (photos) or under permissive cc license (renders)

**Comment by ButzYung** [comment]
I took a quick look at your new sample images and I can tell that some of those photos' licenses are questionable, as they contain renders of ripped data from commercial games (I can identify Dead or Alive, or maybe there are some other games I don't know as well). Images from DeviantART are not reliable all the time lol

I did a search and came across this Pexels website which provides royalty free stock photos of various kind, including some posing models which fits well as samples for `human`.

https://www.pexels.com/

**Comment by vladmandic** [comment]
fair point  
i've removed any cgi images with a questionable source and added 40 stock images from unspash (all with cc license)
